pom.xml

```diff
@@ -0,0 +1,106 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>eclipse.asciidoctor.editor</groupId>
+  <artifactId>releng</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
```

Comment:
Could you please change to 2.5.0-SNAPSHOT or 2.4.2-SNAPSHOT ? Current version is 2.4.1 on eclipse marketplace.

Reply:
Will do but IMHO this is not really important, as this pom file will not create artifacts.